hw/dv/tools/dvsim/testplans/racl_testplan.hjson

```diff
@@ -0,0 +1,36 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: racl_rw
+      desc: '''
+          Precondition:
+            - Randomize the number of RACL policies and randomize its default policy values.
+            - Randomize the policy selection vector of the CSRs.
+            - Randomize the ErrorRsp parameter.
+            - Shuffle the list of CSRs first to remove the effect of ordering.
+
+          Verify the correct access for each CSR:
+            - Loop through each CSR and perform a read and write request using a
+              random RACL role and random CTN UID value.
+              If the request is allowed:
+                - No TL-UL error is expected.
+                - Verify that reads and writes complete as expected.
+              else if the request is denied:
+                - Verify the error log matches:
+                  - error_log_o.valid = 1
+                  - error_log_o.overflow = 0
```

Comment:
Worth testing the case of an overflow as well?

Reply:
Not part of the smoke test, I don't think. This flag means "the error arbiter saw two errors at once". But maybe we should be adding a V2 item to the testplan called something like racl_overflow or maybe racl_arb (if there's anything else sensible we could be checking about the arbiter).

Reply:
A simple IP, lets say UART, only has a single TLUL interface, so there can only one access at a time. Thus, the overflow won't ever be triggered. Overflow can only happen on IPs with multiple interfaces that support RACL, i.e., the sram_ctrl. There the overflow can happened if in the same cycle a register and the memory is accessed and both trigger a RACL violation. I would not add that to smoke.